### DISCUSSION TITLE
linguist-language=py and linguist-vendored not working

### DISCUSSION THREAD

**Discussion by albertbayazidi** [github-linguist/linguist#6235] [Classification] [answered]
I have tried to get linguist-vendored to work. I know that caching is used and changes take a while to get uploaded.Does it have to do with the fact that I don't have ruby? is ruby even nessacery for the manual overrides? I have also tried .ipynb linguist-language=py  to no success. Also tried to reupload the whole repo

was expecting that python was the only langue used in this [repo](https://github.com/albertbayazidi/Prosjekt_Biofysikk) and this [repo](https://github.com/albertbayazidi/Prosjekt_Teknisk_fysikk) after implementing .gitattributes

I have looked at this issue and done the same things [issues 3737](https://github.com/github/linguist/issues/3737)

**Comment by lildude** [comment, marked as answer]
Your overrides aren't working because you're using the wrong filename. The file must be called `.gitattributes`, not `Prosjekt_Biofysikk.gitattributes`.

**Reply by lildude** [reply]
You'll also need to push a change to another file to trigger a re-analysis to take this into account.
